src/MyTested.AspNetCore.Mvc.TempData/Builders/Contracts/Data/IWithoutTempDataBuilder.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace MyTested.AspNetCore.Mvc.Builders.Contracts.Data
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Used for building <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.
+    /// </summary>
+    public interface IWithoutTempDataBuilder
+    {
+        /// <summary>
+        /// Remove temp data entry by providing its key to the built <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.
+        /// </summary>
+        /// <param name="key">Key of the temp data entry.</param>
+        /// <returns>The same <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.</returns>
+        IAndWithoutTempDataBuilder WithoutEntry(string key);
+
+        /// <summary>
+        /// Remove temp data entries by providing their keys to the built <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.
+        /// </summary>
+        /// <param name="entriesKeys">Keys of the temp data entries to be deleted.</param>
+        /// <returns>The same <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.</returns>
+        IAndWithoutTempDataBuilder WithoutEntries(IEnumerable<string> entriesKeys);
+
+        /// <summary>
+        /// Clear all entities from the built <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.
+        /// </summary>
+        /// <returns>The same <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.</returns>
+        IAndWithoutTempDataBuilder WithoutAllEntries();
```

Comment:
There is no need for the word `All` in this method name - make it `WithoutEntries()` only. It is readable and explanatory enough. Update all other Without methods to have the same convention.